DEVELOPER.md

```diff
@@ -1,188 +1,81 @@
 - [Developer quickstart](#developer-quickstart)
-  - [Code style](#code-style)
-  - [Backend setup](#backend-setup)
-  - [Frontend setup](#frontend-setup)
-  - [Setup using Makefile](#setup-using-makefile)
+  - [Running the project](#running-the-project)
+  - [Useful commands](#useful-commands)
+  - [Environment variables](#environment-variables)
+    - [Setting environment variables](#setting-environment-variables)
+    - [`COMPOSE_PROFILES`](#compose-profiles)
   - [Slack application setup](#slack-application-setup)
   - [Update drone build](#update-drone-build)
-- [Troubleshooting](#troubleshooting)
-  - [ld: library not found for -lssl](#ld-library-not-found-for--lssl)
-  - [Could not build wheels for cryptography which use PEP 517 and cannot be installed directly](#could-not-build-wheels-for-cryptography-which-use-pep-517-and-cannot-be-installed-directly)
-  - [django.db.utils.OperationalError: (1366, "Incorrect string value ...")](#djangodbutilsoperationalerror-1366-incorrect-string-value-)
-  - [Empty queryset when filtering against datetime field](#empty-queryset-when-filtering-against-datetime-field)
-- [Hints](#hints)
```

Comment:
Sections under "Hints"/"Troubleshooting" should no longer be relevant now that things are run all in docker